### ISSUE TITLE
Activity Stream should be useable in 'keyboard only' usage

### ISSUE THREAD

**Issue by tspurway** [mozilla/activity-stream#3360]
This appears to be broken currently.

Tabbing does not navigate to every AS UI component.

Hitting <return> on a card, opens it's menu, but arrow keys do not navigate in the menu.  Hitting <return> again gets rid of the menu.  Hitting <esc> does nothing.  There is no way to navigate to the card's URL.

**Comment by dmose** [comment]
@tspurway if you're testing this on Mac, be sure that System Preferences -> Keyboard -> Shortcuts -> "Full Keyboard Access" is set to "all controls" (not the OS default).

**Comment by Mardak** [comment]
Tab moves within the context menu but not arrow keys.

**Comment by tspurway** [comment]
Oh, thanks, @dmose for the tip.  That helps a bit, but 'esc', arrows and the gear icon/preferences panel don't seem to work

**Comment by Mardak** [comment]
#2772 exists for preference pane keyboard accessibility
#2998 has an item "arrow up/down is also expected for menus like that"

**Comment by tspurway** [comment]
OK, @Mardak - so this is a dupe, mostly.  We had put the (keyboard) accessibility at P2 before, so I guess we should stick with this.  I'll close this and decorate those other two bugs with anything else
